src/test/skript/tests/syntaxes/effects/EffHealth.sk

```diff
@@ -8,12 +8,10 @@ test "health effect":
 	spawn zombie at location(0, 64, 0, world "world")
 	set {_m} to last spawned zombie
 	assert health of {_m} is 10 with "default zombie health failed"
-	damage {_m} by 2
-	assert health of {_m} is 8 with "damage zombie ##1 failed"
 	damage {_m} by 0.5
```

Comment:
Why does this change make tests pass? Is there a bug in test system or EffHealth?

Reply:
No, this even happens when using Skript + Skript-mirror. `entity.damage(num)` ... Damaging an entity in quick succession skips the 2nd damage when there is three. Im unsure why, but I would assume this is either a Spigot/CB or MC bugs/glitch.

Reply:
Call me crazy, but what if Skript fixed this issue by calling ``entity.damage()`` AND ``setHealth()``? That way:

- Health is properly reported back to the user in Scripts & Java CI
- Entity will get the damaged visual effect

Even though this may not strictly be a Skript issue, we can still workaround the issue to make sure that users aren't affected by the lack of entity damage visual effects as well as incorrect health being returned. Performance-wise, I don't think this would add much overhead either (correct me if I'm wrong here).

Reply:
Sorry for inactivity. Minecraft has invulnerability ticks after taking damage, probably affects Bukkit API too. This is not a bug, so the changing test makes sense.

@Wealthyturtle We could explore this in later release for 2.5. I'm a bit wary of it due to potential differences in MC versions supported by us.

Reply:
Alright, no problem.